### ISSUE TITLE
Jetpacks compat for sets

### ISSUE THREAD

**Issue by McKitten** [SilentChaos512/Silent-Gear#116]
## What Should Be Added or Changed?

I can see 2 ways to do this one the first is probably the least problematic :
1)  An additional upgrade item (1.12 had a set) of armor plating that could be added to a jetpack so it carries the set bonus of that plating
2) The armor and the jetpack be crafted together side by side. This might cause issues for upgrading the jetpacks tier (current max of 6) but it also could be locked so only at level 6 that it can be added.

## Why Should This Be Done?

Basically I am sick of having to choose between fire ward and my jetpack in the nether, or even trying to repair my monster masher when elite mobs/blazes spawn. There may be others who feel the same about other set bonus (yet to see what icy aura does to a blaze).

Basically ditch the given armor from that tier (is not half bad at 6) and replace it with whatever your set is, maintaining the energy storage & flight speed.

**Comment by SilentChaos512** [comment]
What mod are these jetpacks from? Not sure how much I can do about this, but I'll take a look at it.

**Comment by McKitten** [comment]
Sorry [https://www.curseforge.com/minecraft/mc-mods/iron-jetpacks](url)

and yeah it occurred to me it might need some collaboration, after I wrote this up.
